.github/workflows/samples_configuration.yml

```diff
@@ -9,7 +9,7 @@ on:
     - cron: "40 22 * * *" # Every day starting at 6:40 BJT
   pull_request:
     branches: [ main ]
-    paths: [ examples/**, examples/*requirements.txt, .github/workflows/samples_configuration.yml ]
+    paths: [ examples/**, examples/*requirements.txt, .github/workflows/samples_configuration.yml, '!examples/flow/integrations/**' ]
```

Comment:
don't forget to change the workflow generator

Reply:
good catch, just updated those, need @crazygao to take a look